runbot/models/build_config.py

```diff
@@ -830,8 +831,11 @@ def _post_install_commands(self, build, modules_to_install, py_version=None):
     def _perfs_data_path(self, ext='log'):
         return '/data/build/logs/flame_%s.%s' % (self.name, ext)
 
-    def _coverage_params(self, build, modules_to_install):
-        pattern_to_omit = set()
+    def _coverage_params(self, build, modules_to_install, paths_to_omit=None):
+        if paths_to_omit:
+            pattern_to_omit = set(paths_to_omit.split(','))
+        else:
+            pattern_to_omit = set()
```

Comment:
Hello. 

Why not using self.path_to_omit here? 

```suggestion
    def _coverage_params(self, build, modules_to_install, paths_to_omit=None):
        pattern_to_omit = set()
        if self.paths_to_omit:
            pattern_to_omit = set(self.paths_to_omit.split(','))
```

Reply:
good idea, I have revised as you suggested